bindings/go/src/fdb/tuple/tuple.go

```diff
@@ -314,11 +369,50 @@ func (p *packer) encodeTuple(t Tuple, nested bool) {
 // call Pack when using a Tuple with a FoundationDB API function that requires a
 // key.
 func (t Tuple) Pack() []byte {
-	p := packer{buf: make([]byte, 0, 64)}
+	p := newPacker()
 	p.encodeTuple(t, false)
 	return p.buf
```

Comment:
This won't throw an error (or panic) if the tuple contains an incomplete versionstamp. I think the most idiomatic thing would be to return an error (like in `PackWithVersionstamp`), but that would be API breaking. Maybe panicking isn't so bad? I'm not super fluent on how go people feel about that.

Reply:
Attempting to `Pack` a tuple with a versionstamp using this function is not valid and it would also break the API as you said. The calling code presumably knows what is in the tuple and would call the appropriate function.

Reply:
Hm, so if the user provides an incomplete versionstamp, then I think this would encode it as, what, a versionstamp of all `0xff`s. I think I'd prefer this panic rather than trust the user not supply invalid data.

Reply:
Still yet to address this one.

Reply:
Also, this does not appear to be the behavior of the Python bindings.

Reply:
Done now